### DISCUSSION TITLE
LdapEntry.Attribute is obsolete

### DISCUSSION THREAD

**Discussion by Jamtastic808** [flamencist/ldap4net#153] [Q&A]
Hi
LdapEntry.Attribute is flagging with a warning that it is obsolete.  Should I be worried and what is the proper way of accessing the value of the attribute please? Thank you

**Comment by ckapusta** [comment]
I'm wondering about this too. Can't find any documentation about this. In fact, the readme file still shows examples using it. Feedback would be helpful! @flamencist
